go.graphics.android/src/main/java/go/graphics/android/GOSurfaceView.java

```diff
@@ -191,7 +191,7 @@ public void onTouchEvent(MotionEvent e) {
 					} else if (e.getAction() == MotionEvent.ACTION_UP) {
 						endPan(computePanPoint(e));
 						if (doZoom) {
-							endZoomEvent(factor);
+							endZoomEvent(factor, null);
```

Comment:
null is intended in any cases where zoomEvent is called without a known mouse or pointer position